vabashyn/04_CSS/zadanie3.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
We use html5

Reply:
Sorry but I do not understand what this file is.My repository does not have this file.

Reply:
Could you please remove  xmlns="http://www.w3.org/1999/xhtml"